zerver/webhooks/insping/doc.md

```diff
@@ -0,0 +1,48 @@
+Insping (stylized as !nsping) is a simple uptime and performance monitoring
+tool. You'll get to know when the website or service is up and running or down.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-stream-name.md!}
+
+{!append-topic.md!}
+
+## Put incoming webhook to Insping
+
+1. Go to [Insping website](https://www.insping.com). If you haven't created an
+   account, sign up for a free account.
+
+2. Create an organization in the
+   [manage section](https://app.insping.com/#manage)
+   if there isn't one and set an URL that you want to monitor. After it's done,
+   it should look something like this:
+   ![](/static/images/integrations/insping/001.png)
+
+3. Navigate to [the integrations](https://app.insping.com/#integrations) section.
+   This should appear:
+   ![](/static/images/integrations/insping/002.png)
+
+4. Press "Create Webhook", located in the "Webhooks" section. A box will show up:
+   ![](/static/images/integrations/insping/003.png)
+
```

Comment:
Replace with:
```
In the **Webhooks** section, click on **Create Webhook**:
```

To refer to buttons or sections in a third party website, we usually bold the text, like `**Create Webhook**`.

Reply:
(refer to https://github.com/zulip/zulip/pull/8006#discussion_r161368392)